docs/advanced/plugin/overview.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Overview
+sidebar_label: Overview
+sidebar_position: 0
+---
+
+## Aragon OSx Plugins
+
+As mentioned earlier, plugins built by Aragon and third-party developers can be added and removed from your DAO to adapt it to your needs.
+
+The management of these plugins is handled for you by the Aragon OSx protocol so that the process of
+
+- Releasing new plugins as well as
```

Comment:
Mmm I'm not sure about this one is a little weird having the "as well as" in the same list no?